R/verb-compute.R

```diff
@@ -18,7 +18,7 @@ collapse.tbl_sql <- function(x, ...) {
 
   tbl(x$src, sql) %>%
     group_by(!!! syms(op_grps(x))) %>%
-    add_op_order(op_sort(x))
+    arrange.tbl_lazy(!!!op_sort(x))
```

Comment:
I'm not sure why this used `add_op_order()` instead of `arrange()`. If there is a good reason one can change this to use `add_arrange()`.

Reply:
It doesn't make much sense to me to use either here, since most RDBMs won't order subqueries anyway. I'd recommend leaving it as is ([Chesterton's fence](https://en.wikipedia.org/wiki/Wikipedia:Chesterton's_fence)) and filing an issue to explore in a separate PR.

Reply:
Ah, actually `add_op_order()` comes from `window_order()` not from `arrange()`. I think keeping the `window_order()` makes more sense.

Reply:
If this also makes sense to you, then I change it to `window_order()` and add a test for this.